index.html

```diff
@@ -22,7 +25,22 @@
       //plugins: [
       //  EditOnGithubPlugin.create("https://github.com/exercism/v3/blob/master/")
       //]
+
+      markdown: {
+        renderer: {
+          code: function(code, lang) {
+            if (lang === "mermaid") {
+              return (
+                '<div class="mermaid">' + mermaid.render('mermaid-svg-' + num++, code) + "</div>"
+              );
+            }
+            return this.origin.code.apply(this, arguments);
+          }
+        }
+      }
     }
+    var num = 0;

```

Comment:
Is this needed?

Reply:
It's needed for the name of the svg that mermaid renders. Though I guess I could replace it with some random function if that's better.

Reply:
@SleeplessByte How do we make this less global/leaky?

Reply:
This depends a bit on whether the variable needs to be defined only here or globally at some later time. If it is the former, would an [IFFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) work?

Reply:
That sounds sensible to me.

I'm just wary of it conflicting with some other JS we add later.

Reply:
Replaced it with a random ID.

Reply:
Excellent!